docs/developer/how-to/run-tests.rst

```diff
@@ -11,11 +11,38 @@ It will also report coverage to the commandline and to ``cov.xml``.
 .. _pytest: https://pytest.org/
 .. _look like tests: https://docs.pytest.org/explanation/goodpractices.html#test-discovery
 
-
-Skip the message bus tests
+Configuring message busses
 --------------------------
 
-The tests for stomp require connection to a live ActiveMQ broker and will fail
+The tests for communicating with a message bus over stomp support either ActiveMQ or RabbitMQ
+with the ```rabbitmq_stomp``` plugin installed. The tests assume that such a broker is available
+on ```localhost:61613```: if you require a different host or port, edit StompConfig in config.py.
```

Comment:
```suggestion
on ```localhost:61613```: if you require a different host or port, edit StompConfig in config.py or edit the `BLUEAPI_TEST_STOMP_PORTS` environment variable. This variable is a list of values so the tests can be run against multiple brokers. For example:
```shell
# Default behaviour
export BLUEAPI_TEST_STOMP_PORTS="[61613]"

# Broker is running on a different port
export BLUEAPI_TEST_STOMP_PORTS="[61614]"


# Multiple brokers are running on multiple ports, the tests will be run once per broker
export BLUEAPI_TEST_STOMP_PORTS="[61614]"
```
```